boot-cloverage/src/cloverage/boot_cloverage.clj

```diff
@@ -0,0 +1,36 @@
+(ns cloverage.boot-cloverage
+  (:require [boot.core :as core]
+            [boot.pod :as pod]
+            [boot.util :as util]
+            [clojure.string :as s]
+            [clojure.set :as set]))
+
+(def ^:private deps
+  '[[cloverage "1.0.8-SNAPSHOT"]])
```

Comment:
Is there a way to get this from the project.clj automatically so we don't have to repeat this?